_examples/examples/config/gradle/dependencies.gradle

```diff
@@ -0,0 +1,292 @@
+/*
+ * Copyright 2020, TeamDev. All rights reserved.
```

Comment:
I don't understand whether this file is in this repository, or it is grabbed from the `config` submodule, as it should.
Please review.

The same applies to the rest of the files in this folder.